me.simakis.dropboxignore.desktop

```
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=dropboxignore
+Exec=/app/bin/dropboxignore.sh
```

```suggestion
Exec=dropboxignore.sh
```

it's never an absolute path